src/modules/dashboard/docs/it_IT.md

```diff
@@ -0,0 +1,45 @@
+## Riepilogo
+
+Il riepilogo ti darà una rapida panoramica degli edifici e veicoli.
+
+![Riepilogo](summary.png)
```

Comment:
Pictures will not be shown. You will need to put them in the according directory (`/src/modules/{module}/docs/assets/it_IT`).
You may copy pictures from other languages or create own italian screenshots.
Same goes for any of the other pictures.

Reply:
I solved this specific problem but I noticed other assets that weren't in it_IT so I need to correct all others assets that weren't made?

Reply:
Yes, all referenced pictures need to be within the it_IT folder as a fallback to en folders does not exist.

Reply:
So now I added all the assets to all modules that needed that